integration-test/helpers/mocks.webkit.js

```diff
@@ -241,6 +293,49 @@ export function createWebkitMocks (platform = 'macos') {
         withFeatureToggles: function (_featureToggles) {
             throw new Error('unreachable - webkit cannot mock feature toggles this way. Use script replacements')
         },
+        withCheckCredentialsProviderStatus: function () {
```

Comment:
to get some nice TS help whilst authoring, can you the type of this array? 

Something like `/** @type {import('./foo'').Bar[]} */` with the import pointing to the schema types.